epictrack-api/src/api/resources/__init__.py

```diff
@@ -105,3 +105,5 @@
 API.add_namespace(RESPONSIBILITY_API, path="/responsibilities")
 API.add_namespace(OUTCOME_CONFIGURATION_API, path="/outcome-configurations")
 API.add_namespace(ACT_SECTION_API, path="/act-sections")
+API.add_namespace(WORK_STATUS_API, path='/work/<int:work_id>/status')
```

Comment:
Do you think it would be good to have plurals in the URL like statuses instead of status

Reply:
done

Reply:
changed to statuses